assets/src/components/account/groups/Group.tsx

```diff
@@ -30,29 +34,21 @@ export default function Group({ group, q }: { group: GroupT; q: any }) {
   const [mutation, { loading, error }] = useDeleteGroupMutation({
     variables: { id: group.id },
     onCompleted: () => dialogKey === 'confirmDelete' && setDialogKey(''),
-    update: (cache, { data }) => updateCache(cache, {
-      query: GroupsDocument,
-      variables: { q },
-      update: prev => removeConnection(prev, data?.deleteGroup, 'groups'),
-    }),
+    update: (cache, { data }) =>
+      updateCache(cache, {
+        query: GroupsDocument,
+        variables: { q },
+        update: (prev) => removeConnection(prev, data?.deleteGroup, 'groups'),
+      }),
   })
 
   return (
-    <Flex
-      width="100%"
-      flexDirection="row"
-      alignItems="center"
-    >
+    <Flex width="100%" flexDirection="row" alignItems="center">
       <Info
         text={group.name}
         description={group.description || 'no description'}
       />
-      <Flex
-        flex={false}
-        direction="row"
-        gap="large"
-        align="center"
-      >
+      <Flex flex={false} direction="row" gap="large" align="center">
```

Comment:
One thing to consider is keeping args in separate lines. I think it was easier to read. What do you think?

Reply:
Agree, I also prefer arg-per-line style.

Reply:
Also makes for cleaner diffs. Looks like we all agree here!